Custom chips

FaizanAmin2266 opened the pull request:
Custom chips component with the different styles

FaizanAmin2266 committed:
test

FaizanAmin2266 committed:
save

FaizanAmin2266 committed:
Merge branch 'develop' of https://github.com/gopher-tunnels/gt-front-end into custommarkers

FaizanAmin2266 committed:
save

FaizanAmin2266 committed:
save

FaizanAmin2266 committed:
save

FaizanAmin2266 committed:
Merge branch 'develop' of https://github.com/gopher-tunnels/gt-front-end into custommarkers

FaizanAmin2266 committed:
s

FaizanAmin2266 committed:
s

FaizanAmin2266 committed:
tfdhrth
\

limetheman reviewed (changes requested):
Hey faizan! Great first PR, I just have some comments for more food for thought on your implementation mostly about how we are passing in values

limetheman commented:
Also minor comment: With your commit messages, make sure they are descriptive to what you were doing at the time and they are in present tense

FaizanAmin2266 committed:
Chips use theme colors.
Width and height are no longer hardcoded.
Chips prop renamed to CustomChipsProp.
Unused comments removed.
Chips can take in array with multiple names and styles.
Duplicate CustomChips removed.

FaizanAmin2266 commented:
I updated CustomChips branch with the changes

VicHofs committed:
Rename 'CustomChips' to 'CustomChip';
Remove Chips from Home screen

VicHofs committed:
Merge branch 'develop' into CustomChips
